### ISSUE TITLE
Is CuDNN 5 supported? 

### ISSUE THREAD

**Issue by AlexTS1980** [torrvision/crfasrnn#97]
Is CuDNN 5 supported? I installed vanilla caffe w/out any problem, but crf-rnn caffe keeps telling me 

./include/caffe/util/cudnn.hpp: In function ‘void caffe::cudnn::createPoolingDesc(cudnnPoolingStruct**, caffe::PoolingParameter_PoolMethod, cudnnPoolingMode_t*, int, int, int, int, int, int)’:
./include/caffe/util/cudnn.hpp:124:41: error: too few arguments to function ‘cudnnStatus_t cudnnSetPooling2dDescriptor(cudnnPoolingDescriptor_t, cudnnPoolingMode_t, cudnnNanPropagation_t, int, int, int, int, int, int)’
         pad_h, pad_w, stride_h, stride_w));

**Comment by bittnt** [comment]
Yes, you should check https://github.com/BVLC/caffe/pull/4159 I think I have merged this with a branch in caffe-crfasrnn.

**Comment by AlexTS1980** [comment]
What changes do I need to make?
